spectrum/autism_prevalence_map/templates/autism_prevalence_map/base.html

```diff
@@ -20,84 +20,49 @@
     <meta property="og:image" content="https://prevalence.spectrumnews.org/static/autism_prevalence_map/img/PrevalenceMapSG.png" />
     <meta property="og:description" content="The map features a collection of studies on autism prevalence around the world. It highlights places where information is available — and places where information is missing." />
 
-    <!-- Mobile Specific Metas
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <!-- CSS libraries
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <!-- <link rel="stylesheet" href="https://use.typekit.net/hjy6pmo.css"> -->
-    <script src="https://use.typekit.net/gnz4dab.js"></script>
-    <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/vendor/css/bootstrap.min.css' %}">
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
-    <!-- Custom site-wide CSS
-    –––––––––––––––––––––––––––––––––––––––––––––––––– -->
-    <link rel="stylesheet" type="text/css" href="{% static 'autism_prevalence_map/css/style.css' %}">
+    {% static '' as static_base %}
```

Comment:
To be used as a variable in the following line and down on L115
It's basically a way of casting the statics location to a simple variable so it can be concatenated with another variable (like the CSS or JS file names)